client/src/app/hooks/table-controls/useLocalTableControls.ts

```diff
@@ -34,16 +34,17 @@ export const useLocalTableControls = <
   const state = useTableControlState(args);
   const derivedState = getLocalTableControlDerivedState({ ...args, ...state });
   const { columnState } = state;
+  const selectionState = useSelectionState({
+    ...args,
+    isEqual: (a, b) => a[args.idProperty] === b[args.idProperty],
+  });
+
   return useTableControlProps({
     ...args,
     ...state,
     ...derivedState,
-    // TODO we won't need this here once selection state is part of useTableControlState
```

Comment:
I know those comments are a bit outdated but I would keep them because the area is complex